pgmpy/factors/CPD.py

```diff
@@ -116,15 +119,33 @@ def __init__(self, variable, variable_card, values,
 
         super(TabularCPD, self).__init__(variables, cardinality, values.flatten('C'))
 
-    def get_cpd(self):
+    def get_cpd(self, pretty=True):
         """
         Returns the cpd
         """
         if self.variable in self.variables:
+            if pretty:
+                table_list = []
+                for i in range(len(self.cardinality))[1:]:
```

Comment:
Would be better to simply use `range(1, len(self.cardinality))` or range(len(self.cardinality) - 1)`. It would be much faster and also is more readable.